sound/soc/sdw_utils/soc_sdw_rt721_sdca.c

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: GPL-2.0-only
+// This file incorporates work covered by the following copyright notice:
+// Copyright (c) 2024 Advanced Micro Devices, Inc.
+
+/*
+ *  soc_sdw_rt721_sdca - Helpers to handle RT721-SDCA from generic machine driver
+ */
+
+#include <linux/device.h>
+#include <linux/errno.h>
+#include <linux/soundwire/sdw.h>
+#include <linux/soundwire/sdw_type.h>
+#include <sound/control.h>
+#include <sound/soc.h>
+#include <sound/soc-acpi.h>
+#include <sound/soc-dapm.h>
+#include <sound/soc_sdw_utils.h>
+
+static const struct snd_soc_dapm_route rt721_spk_map[] = {
+	{ "Speaker", NULL, "rt721 SPK" },
+};
+
+int asoc_sdw_rt721_spk_rtd_init(struct snd_soc_pcm_runtime *rtd, struct snd_soc_dai *dai)
```

Comment:
Can we merge those Realtek multi-function codecs spk_rtd_init? Like what we did in soc_sdw_rt_amp.c. Currently, we have soc_sdw_rt712_sdca.c, soc_sdw_rt722_sdca.c, and soc_sdw_rt721_sdca.c, and I am sure there will be more coming.

Reply:
Agree, shall respin patch..